src/key.rs

```diff
@@ -284,8 +295,8 @@ impl Key {
 
     pub fn to_bytes(&self) -> Vec<u8> {
         match self {
-            Key::Public(k) => k.to_bytes().unwrap_or_default(),
-            Key::Secret(k) => k.to_bytes().unwrap_or_default(),
+            Key::Public(k) => Serialize::to_bytes(&k).unwrap_or_default(),
+            Key::Secret(k) => Serialize::to_bytes(&k).unwrap_or_default(),
```

Comment:
These are the same `to_bytes()` calls as before, but because the `DcKey` trait now also has a `to_bytes()` method and is active since it's defined in this file, we have to explicitly call the version of rpgp's `Serialize` trait.